### PR TITLE
Added Header

### PR DESCRIPTION
**Pull request by Hariprasad19036**
I've added Header to the portfolio. Main and Nav tags were already added wherever necessary so I skipped it. LMK if anything else must be added.

### CODE REVIEW

File: src/components/layout/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Header = () => {
+    return (
+        <div className='header'>
```
Comment (EmmaDawsonDev) [inline src/components/layout/Header.jsx:5]:
```suggestion
        <header className='header'>
```

This is the same kind of change that needs making to the other components as well. Find the divs marking the nav, main and footer and replace the divs with the semantic version of the tag.

**Comment by sano2019** [comment]
Hi! This isn't really what I was intending. I would like to make the site more accessible by using more semantic html elements. In the case where you are adding a new component, it is still a regular DIV with a class of header. If you are interested in learning more about semantic html, and it's use in accessibility, I suggest having a look here:

http://web-accessibility.carnegiemuseums.org/foundations/semantic/#:~:text=Structural%2C%20semantic%20HTML%20is%20the,read%20by%20a%20screen%20reader.

There are also some resources linked there. This is kind of what I am looking for, so that the site becomes more accessible also for people using screen readers etc. Let me know if you have any questions :)

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Hi @Hariprasad19036 There's a couple of small changes to make before this can be merged. See the comment about semantic tags. Also, please remove the yarn.lock file from this pull request.

**Comment by Hariprasad19036** [comment]
Thanks for clarifying! Will look into it.

**Pull request closed without merging**